### ISSUE TITLE
Not registering post type inside plugin

### ISSUE THREAD

**Issue by manuman94** [TypeRocket/typerocket#93]
Hi, 
I'm trying to register the example "Book" post type at my plugin. I installed Typerocket and I'm including it at the plugin. The "dev" tab shows at admin panel but I can't register post types. 

I'm var_dumping the variable and shows that the method is executing, but is not showing at the admin panel.

How can I get it to work? 

Thank you, Greetings.

**Comment by manuman94** [comment]
UPDATE: It's registering it if I use it in my template, but not at my plugin.

**Comment by kevindees** [comment]
Hey @manuman94 

What does your plugin code look like?

Thanks,
Kevin

**Comment by manuman94** [comment]
Hi! 

First, thank you for Typerocket and for this support. It's awesome.

My plugin has at the root file a "init" function that initializes all what it does. It declare all the actions, filters, and so on. To make Typerocket work (following its docs) I implemented the typerocket_loaded action (that never get triggered). Then, I tried to register the example "book" outside the "typerocket_loaded" and code was arriving there (var dump showed the class information), but the post type wasn't showing at admin panel.

Then I tried to implement it at the template and it got working quickly. 

The plugin is so long, that's why I don't share it here.

Thank you, greetings.
